modules/react/menu/lib/MenuItem.tsx

```diff
@@ -180,9 +294,9 @@ export const MenuItem = createSubcomponent('button')({
 })<MenuItemProps>(({children, ...elemProps}, Element) => {
   return (
     <OverflowTooltip placement="left">
-      <StyledItem minHeight={space.xl} as={Element} {...elemProps}>
```

Comment:
NB: I can't comment this, but we can probably use our new text component here: https://github.com/Workday/canvas-kit/blob/master/modules/react/menu/lib/MenuItem.tsx#L178